packages/vite/src/node/watch.ts

```diff
@@ -10,9 +10,27 @@ export function resolveChokidarOptions(
 
   const resolvedWatchOptions: WatchOptions = {
     ignored: [
-      '**/.git/**',
-      '**/node_modules/**',
-      '**/test-results/**', // Playwright
+      ...[
+        '.git/**',
+        '.git*',
+        '.github/**',
+        '.vscode/**',
+        '.stackblitz/**',
+        'node_modules/**',
+        'test-results/**', // Playwright
+        'coverage/**',
+        '.editorconfig',
+        '.eslint*',
+        '.prettier*',
+        '.npmrc',
+        '.yarnrc',
+        '*.toml',
+        '*.yaml',
+        '*.md',
```

Comment:
These are valid files to import in a normal project, we shouldn't include them here.